pkg/virt-operator/resource/apply/reconcile_test.go

```diff
@@ -22,9 +22,7 @@ package apply
 import (
```

Comment:
Please add a functional test for your use-case

Reply:
Done. Please see the latest commit.